clang/docs/ReleaseNotes.rst

```diff
@@ -105,6 +105,11 @@ C2y Feature Support
 C23 Feature Support
 ^^^^^^^^^^^^^^^^^^^
 
+Non-comprehensive list of changes in this release
+-------------------------------------------------
```

Comment:
https://github.com/llvm/llvm-project/commit/10c6d6349e51bb245b9deec4aafca9885971135b#r145075257

Looks like this was removed by accident. We should probably wait for a separate NFC patch to add this so there's no chance of it getting reverted again

Reply:
Opened as #102134